examples/src/main/scala/caliban/akkahttp/ExampleApp.scala

```diff
@@ -0,0 +1,56 @@
+package caliban.akkahttp
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.stream.ActorMaterializer
+import akka.http.scaladsl.server.Directives._
+import caliban.{AkkaHttpAdapter, ExampleService, RootResolver}
+import caliban.ExampleApp.{Mutations, Queries, Subscriptions}
+import caliban.ExampleData.sampleCharacters
+import caliban.GraphQL.graphQL
+import zio.DefaultRuntime
+
+import scala.io.StdIn
+
+object ExampleApp extends App {
+
+  implicit val system = ActorSystem()
+  implicit val materializer = ActorMaterializer()
+  implicit val executionContext = system.dispatcher
+  implicit val defaultRuntime = new DefaultRuntime {}
+
+  case class Character(name: String)
```

Comment:
How about reusing `ExampleData` and `ExampleService` so that this example is easily comparable to the http4s one?

Also you could split the current `ExampleApp` to move the http4s part into an `http4s` folder (just like you did for akka) and keep the common part in the shared folder (Queries, etc).

Reply:
resolved in 0cbe7a7 and 430c3c9